packages/example-app-base/src/components/ComponentPage/ComponentPage.tsx

```diff
@@ -356,6 +364,24 @@ export class ComponentPage extends React.Component<IComponentPageProps, {}> {
     return undefined;
   }
 
+  private _getFeedback(): JSX.Element | undefined {
+    if (this.props.feedback) {
```

Comment:
This is not very intuitive. What do you anticipate most callers (I.e. each of the example pages) to pass in here?